### PR TITLE
br/stream: make status shows the log

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb/issues/30544

Problem Summary:

This PR:
1. Make `stream status` shows information about last errors.
2. Clean the error when executing `stream resume`.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/stream/client.go [hunk 1]
```diff
@@ -85,6 +87,14 @@ func (c *MetaDataClient) ResumeTask(ctx context.Context, taskName string) error
 	return nil
 }
 
+func (c *MetaDataClient) CleanLastErrorOfTask(ctx context.Context, taskName string) error {
```
Comment (3pointer) [inline br/pkg/stream/client.go:90]: what if this error cannot be resume 🤔

File: br/pkg/task/stream.go [hunk 2]
```diff
@@ -720,6 +720,11 @@ func RunStreamResume(
 	if err != nil {
 		return errors.Trace(err)
 	}
+
+	err = cli.CleanLastErrorOfTask(ctx, cfg.TaskName)
```
Comment (3pointer) [inline br/pkg/task/stream.go:724]: what if the error cannot be resumed?🤔

Comment (YuJuncen) [reply br/pkg/task/stream.go:724]: Maybe it would pause again?

**Comment by YuJuncen** [comment]
image

**Review by joccau — APPROVED** [review]
LGTM

File: br/pkg/stream/stream_status.go [hunk 3]
```diff
@@ -152,15 +167,20 @@ func (p *printByJSON) PrintTasks() {
...
-		EstQPS      float64         `json:"estimate_qps"`
+		Name        string           `json:"name"`
+		StartTS     uint64           `json:"start_ts,omitempty"`
+		EndTS       uint64           `json:"end_ts,omitempty"`
+		TableFilter []string         `json:"table_filter"`
+		Progress    []storeProgress  `json:"progress"`
+		Storage     string           `json:"storage"`
+		Checkpoint  uint64           `json:"checkpoint"`
+		EstQPS      float64          `json:"estimate_qps"`
+		LastError   []storeLastError `json:"last_errors"`
```
Comment (3pointer) [inline br/pkg/stream/stream_status.go:183]:
```suggestion
		LastErrors   []storeLastError `json:"last_errors"`
```

**Review by 3pointer — APPROVED** [review]
rest LGTM

**Pull request merged**